fixieai/agents-ts/src/fixtures/normal/chart-data.txt

```diff
@@ -0,0 +1 @@
6nOzirkiOY5jQ2k93Voqm2cLFixYABWPAcC/pJTXIaS+Xozm5uYMER0NVRawju/7k4NzlmWdCWWPHWXmv6Hg/yGZTEYty5oBIEJEM1fhpfR7ukzF3fATQogHoOrPP0wkEtsWrj4ZDAaDwWAwdAe2be9PRHcB+JGIJriue2/4fENDQ6LYc4pOeW2BenYcAWWjezqAAUQ0o729PdnU1PRt0F4IURGJRBbPmjWrLXTMg6p7fRfA9lLKbJdcpKHPMXny5IELFix4B8BGzHwbEe0IIAi6/+n7/mnpdPqJwn6O49zKzIcBuENKOTY4PnHixPJ8Pv8SgM0BtDJz3bfffnv/oEGDhsRiseMBnATgP9lsdo9Ssi22bXsMEd0NAET0B9d1H1iRfkKIU6CE475LJBK/CASsHcf5EzM3Q23q3mNZ1ulDhw6d29raOoqZzwewOxF5ruuKLrqkfkmX+KAbloSIjgPQDmDT9vb2yctrbzAYDAaDwbCqqa2tXY+IbgWQYebdCoNzAOhsE0GnvAY74y8AuJCZX7EsawfXdY8vCM7PBzAvl8sND4+RzWZPg9qR24qZj1tFl2UoAQqyTkdls9mdATQA+BzALpZlPW7b9iOO4+wY7heJRE4AsADAn4QQvw2Oz5o1q82yrP0BzAFQQUR3l5eXL47FYl8AOI2IHrIsa89SCs4BQAvmPQgAzHxFQ0NDYkX6JRKJmQC+B7Dm4sWL9wmOu657BzNXQ9Wgj/F9/62WlpYsMz8NZeN3jAnOVz0mQO8GXNf9AMCV+uWZQohf9OR8DAaDwWAw9D8sy6oDsCaA81amFJGZL9c/Jpj5PM/zfpNKpeYUaboLAPJ9f174oA6GzgIAIjpLCFHxv87BULp4nncnlNJ/ZTQaPVdKeTWATQCcCqCNiPZj5pdt224WQmwJANot4Dw9xJVVVVXxYLxUKvVxWVnZzgCO03XWbzDzNZZl/cZ13YMKhRBLBWY+HkqcerPFixdPWZE+06dP/56IngEAIhoRPud53uxIJLItgMuI6GGtNXYigI2klDNW9fwNJsW926ivrx+kLSGGMfPNnucd0dNzMhgMBoPB0H9wHOcOZj6EmWs8z7ums3YTJkxYPR6PD2fmIb7vf+p53r8BcDKZtObNm/cMEe0G4ON8Pj+i0GNbp9jeBeApKeXehWNXVVVFysvLXwWwPYAbpJTjVvFlGvowjuNsxsxvAYhZljUylUq9DKj3ZCwWq2fm0wCsAaVncLvv+yevv/76n7a2tr7EzCMAnCSlvKQHL6FXIIS4EMApABZEo9GtAtvD5fQ5G0ASwC1Syr909RwNnWN20LsJLUBxEoAfiOjVnp6PwWAwGAyG/gUztwAAER2dTCY7ngGTyaTlOM6OQojTHcd5MRqNzvd9/zlmvoeIXrdt+6WJEydukUwmfS0o9Q2AjSKRyONCiF0AVfNr2/ZZRHQ7gBbLsooG3s3NzXnf9xugxKvGaBVogwHAElmnlu/7M4L36TXXXPOD67oXA9iEmS+G2iGusixrbktLywwduGcBnFlXVzesxy6glxCNRs8H8BmAsnw+f+mK9GHmQFBuUZdNzLBCmB307oXq6urWKdWUGoPBYDAYDL0Xx3F21HZnBOA1AE8CWB/APgDWXk73/wAYLqVcaNv2TlqIKkhR/wHAagAiAB7xff+o5SmzCyEcAPdIKVtX/ooMpUhB1mnRbI+ampoNI5HI2QCOgnrffQ8VWK5HRDe6rntk986692Hb9p+J6Gb9cm8p5ZPLai+EuAbAeCKa6rru5ctqa+haTIBuMBgMBoPB0E9wHOccba9WjE+Y+R4AD+ZyuVd+/PHH+UOGDDmEmW+Aqjuf5HneTAAYP378OrFY7DgAexNRnJnnMPMNnuc91l3XYihdhBBHALgRwJeZTGaLsAhhQbstAZwL4FD8FNf8CGBzs/gD2Lb9JBHtBeUZv0NnzglCiNUAfApgjXw+v2ljY+Mn3TlPw5KYAN1gMBgMBoOhH+E4zkEAjmTmIcz8FRE9zcxPdiYcJ4S4GMBJzHyb53lV3TtbQz+FhBBPAdgTwFVSymWq/gshfg0lFve97/snpNPpz7pjkr2durq6bX3ffx1AFMBkKeUVxdo5jiOZ2QYwTUq5QsJyhq7DBOgGg8FgMBgMhk6xbXssEd1mAnRDd2Lb9jZE9AYAIqIRruv+u6fn1BdxHOcKbWv4ne/7W4TLTxoaGhKZTOZyZq6Hsk/cq7NddkP3YUTiDAaDwWAwGAzLYhQAWJb1eE9PxNB/8DzvbWaeBSDCzFfDbCyuFMx8NoAvAKxpWdbFwXHbtke3t7e/ooPzW3K53O9McN47MG90g8FgMBgMBkNRHMfZh5kfAvBRIpEYPn369PaenpOh/9DQ0LBGe3v7ewCGEtERruvevNxOhqWwbbuaiK6FEmuvAjCWiP4M4CtmPtnzvKYenqIhhAnQDQaDwWAwGPo3BGV7tsQxx3GOYeZLoFJj90qn0+/2wNwM/RzHcWqYOQ3g80QiscX06dO/7+k59UHIcZwXmHlX/dqHEuGbIqX8ugfnZShCtKcnYDAYDAaDwWDoGRzHEcxcR0STKioq/vXpp5+uEY1G92Pmk5l5ByJ6NZ/PV6XT6f/29FwN/ZOKiopr582bJ4jol+3t7VsB+GdPz6kPwsw8CcDLRPRCPp8/Jp1Ov9nTkzIUx+ygGwwGg8FgMPRDHMfZmplfBxDXh9oBJPTPLQAuBXC1qUs19DSO42yWy+W+aWxsnN/Tc+nLCCF2kVK+jKUzZgy9CBOgGwwGg8FgMPRTbNv+FRGdDGAXABYzv01EdyYSidmm3txgMBgMBoPBYDAYDAaDwWAwGAwGg8FgMBgMBoPBYDAYDAaDwWAwGAwGg8FgMBgMBoPBYDAYDAaDwWAwGAwGg8FgMBgMBoPBYDAYDAaDwWAwGAwGg8FgMBgMBoPBYDAYDAaDwWAwGAwGg8FgMBgMBoOh5/l/OY4pM3f/wK8AAAAASUVORK5CYII=
```

Comment:
did you try converting this to png? Might be a lot smaller

Reply:
I didn't try to optimize for file size at all. Do you think it matters?